content/docs/ops/runbook/rotating-kubernetes.md

```diff
@@ -9,22 +9,43 @@ title: Rotating Secrets VI - Kubernetes
 
 ## Introduction
 
-Cloud.gov uses kubernetes to provided managed services, including elasticsearch and redis. The kubernetes bosh deployment and service broker are deployed from https://github.com/18F/cg-deploy-kubernetes.
+cloud.gov uses Kubernetes to provided managed services, including elasticsearch
+and redis. The Kubernetes bosh deployment and service broker are deployed from
+https://github.com/18F/cg-deploy-kubernetes.
+
+When rotating Kubernetes, you must open a scheduled maintenance window in
+Statuspage. Use the **Maint - GovCloud - Experimental services disruption**
```

Comment:
Not sure if I should mention StatusPage here since it might be better to just use more generic language since this will be out-of-date once we switch to ServiceNow.

Reply:
Agree. Should probably switch out to something more generic.